webpack/assets/javascripts/react_app/components/notifications/toast/ToastsList.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import Toast from './Toast';
+import * as ToastActions from '../../../redux/actions/toasts';
+import { connect } from 'react-redux';
+import _ from 'lodash';
+
+class ToastsList extends Component {
+
+  render() {
+    const { messages } = this.props;
+
+    const visibleToasts = _.filter(messages, message => {
+      return message.visible;
+    });
+
+    const markup = _.map(visibleToasts, message => {
```

Comment:
nitpick:maybe its me, but i find those extra vars unneeded, i can see one loop over all visible messages that generate their markup inside the return block..